src/components/SearchBar.js

```diff
@@ -54,6 +64,14 @@ type SearchBarProps = {
   buttonLabel: string;
   /** If set, this is the route to navigate to upon executing a search. By default, no navigation will occur when searching. */
   route: string | null;
+  /** If set, the save search option is displayed next to the SearchBar, false by default */
+  allowSavedSearch: boolean;
```

Comment:
New props should be optional for backwards compatibility.
`allowSavedSearch?: boolean;`